cfgov/es_mappings/calendar_event.json

```diff
@@ -0,0 +1,59 @@
+{
+  "properties": {
+    "active": {
+      "type": "boolean"
+    },
+    "all_day": {
+      "type": "boolean"
+    },
+    "calendar": {
+      "type": "string",
+      "index": "not_analyzed"
+    },
+    "created": {
+      "type": "date",
+      "format": "dateOptionalTime"
+    },
+    "date": {
+      "type": "date",
+      "format": "dateOptionalTime"
+    },
+    "day": {
+      "type": "date",
+      "format": "dateOptionalTime"
+    },
+    "description": {
+      "type": "string"
+    },
+    "dtend": {
```

Comment:
This isn't new code-- this is just the old elasticsearch mappings from the old "sheer" version of the site.